seatunnel-e2e/seatunnel-connector-v2-e2e/connector-http-e2e/src/test/java/org/apache/seatunnel/e2e/connector/http/HttpIT.java

```diff
@@ -363,6 +363,39 @@ public void testSourceToAssertSink(TestContainer container)
         Assertions.assertEquals(0, execResult21.getExitCode());
     }
 
+    @TestTemplate
+    public void testPlaceholderReplacement(TestContainer container)
```

Comment:
Let's move all test case to unit test. You can refer https://github.com/apache/seatunnel/pull/9103/files#diff-4926ea699715eac0714ccf51f2093fa36b3d15f0a72239490b4ba77de5674bceR54